src/Fetcher/fetcher.go

```diff
@@ -86,61 +124,72 @@ func errHandler(err error, params ...string) {
 	}
 }
 
-func executer(cmd *exec.Cmd) error {
+func executer(cmd *exec.Cmd, folder string) error {
 	//TODO: Figure out a way to take strings as input and build cmd
```

Comment:
I'm assuming this has been done? 
```suggestion
```

Reply:
This has not been done. This is an old comment pertaining to the fact that the executer should take the string args directly instead of taking the cmd *exec.cmd since it would minimize redundancy but I never figured out a way to do that